inlong-tubemq/tubemq-client/src/main/assembly/assembly.xml

```diff
@@ -18,7 +18,6 @@
 <assembly>
     <id>bin</id>
     <formats>
-        <format>dir</format>
```

Comment:
here we should keep the dir format, because it may be used in https://github.com/apache/incubator-inlong/blob/master/inlong-distribution/distribution.xm to package the whole binary package.